src/shapes/polygon.class.ts

```diff
@@ -1,3 +1,4 @@
+import { IPoint } from '../point.class';
```

Comment:
?

Reply:
probably a leftover of a previous thing.

Reply:
Oh right, when it comes to assigning defaults this will have a constructor too.